cmd/podman/kube/play.go

```diff
@@ -253,19 +253,31 @@ func play(cmd *cobra.Command, args []string) error {
 	}
 
 	if playOptions.Down {
-		return teardown(reader, entities.PlayKubeDownOptions{Force: playOptions.Force})
+		return teardown(reader, entities.PlayKubeDownOptions{Force: playOptions.Force}, false)
 	}
 
 	if playOptions.Replace {
-		if err := teardown(reader, entities.PlayKubeDownOptions{Force: playOptions.Force}); err != nil && !errorhandling.Contains(err, define.ErrNoSuchPod) {
+		if err := teardown(reader, entities.PlayKubeDownOptions{Force: playOptions.Force}, false); err != nil && !errorhandling.Contains(err, define.ErrNoSuchPod) {
```

Comment:
Consider moving the teardown on error into a `defer`.  This way, we make sure that future code changes are covered by the check as well.

Reply:
fixed